assets/src/application/ui/layout/entityList/EntityList.tsx

```diff
@@ -33,7 +33,7 @@ const EntityList = <E extends Entity, ELFS extends EntityListFilterStateManager<
 	const error = isError(entityType);
 	const loading = isLoading(entityType);
 
-	const filteredEntities = useFilteredEntities(domain, listId, entities, filterState);
+	const { paginatedEntities, searchResults } = useFilteredEntities(domain, listId, entities, filterState);
```

Comment:
I am not :100: happy that `useFilteredEntities` is now leaking it's internals, open to suggestions